2024-06-13 19:45:15,241 INFO    MainThread:2419930 [wandb_setup.py:_flush():76] Current SDK version is 0.17.1
2024-06-13 19:45:15,241 INFO    MainThread:2419930 [wandb_setup.py:_flush():76] Configure stats pid to 2419930
2024-06-13 19:45:15,241 INFO    MainThread:2419930 [wandb_setup.py:_flush():76] Loading settings from /home/foottracker/.config/wandb/settings
2024-06-13 19:45:15,241 INFO    MainThread:2419930 [wandb_setup.py:_flush():76] Loading settings from /home/foottracker/myenv/FootTracker/Detection/wandb/settings
2024-06-13 19:45:15,241 INFO    MainThread:2419930 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-06-13 19:45:15,241 INFO    MainThread:2419930 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'sweep-grid.py', 'program_abspath': '/home/foottracker/myenv/FootTracker/Detection/sweep-grid.py', 'program': 'sweep-grid.py'}
2024-06-13 19:45:15,242 INFO    MainThread:2419930 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-06-13 19:45:15,242 INFO    MainThread:2419930 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-06-13 19:45:15,242 INFO    MainThread:2419930 [wandb_init.py:_log_setup():520] Logging user logs to /home/foottracker/myenv/FootTracker/Detection/wandb/run-20240613_194515-a09sl9lc/logs/debug.log
2024-06-13 19:45:15,242 INFO    MainThread:2419930 [wandb_init.py:_log_setup():521] Logging internal logs to /home/foottracker/myenv/FootTracker/Detection/wandb/run-20240613_194515-a09sl9lc/logs/debug-internal.log
2024-06-13 19:45:15,242 INFO    MainThread:2419930 [wandb_init.py:init():560] calling init triggers
2024-06-13 19:45:15,242 INFO    MainThread:2419930 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {}
2024-06-13 19:45:15,242 INFO    MainThread:2419930 [wandb_init.py:init():610] starting backend
2024-06-13 19:45:15,242 INFO    MainThread:2419930 [wandb_init.py:init():614] setting up manager
2024-06-13 19:45:15,245 INFO    MainThread:2419930 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-06-13 19:45:15,246 INFO    MainThread:2419930 [wandb_init.py:init():622] backend started and connected
2024-06-13 19:45:15,252 INFO    MainThread:2419930 [wandb_init.py:init():711] updated telemetry
2024-06-13 19:45:15,283 INFO    MainThread:2419930 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2024-06-13 19:45:15,676 INFO    MainThread:2419930 [wandb_run.py:_on_init():2402] communicating current version
2024-06-13 19:45:15,780 INFO    MainThread:2419930 [wandb_run.py:_on_init():2411] got version response 
2024-06-13 19:45:15,781 INFO    MainThread:2419930 [wandb_init.py:init():795] starting run threads in backend
2024-06-13 19:45:16,181 INFO    MainThread:2419930 [wandb_run.py:_console_start():2380] atexit reg
2024-06-13 19:45:16,181 INFO    MainThread:2419930 [wandb_run.py:_redirect():2235] redirect: wrap_raw
2024-06-13 19:45:16,181 INFO    MainThread:2419930 [wandb_run.py:_redirect():2300] Wrapping output streams.
2024-06-13 19:45:16,181 INFO    MainThread:2419930 [wandb_run.py:_redirect():2325] Redirects installed.
2024-06-13 19:45:16,183 INFO    MainThread:2419930 [wandb_init.py:init():838] run started, returning control to user process
2024-06-13 19:45:16,960 INFO    MainThread:2419930 [pyagent.py:run():323] Starting sweep agent: entity=None, project=None, count=None
2024-06-13 19:45:17,125 ERROR   MainThread:2419930 [internal_api.py:execute():344] 404 response executing GraphQL.
2024-06-13 19:45:17,126 ERROR   MainThread:2419930 [internal_api.py:execute():345] {"errors":[{"message":"could not find project foottracker/uncategorized during createAgent","path":["createAgent"]}],"data":{"createAgent":null}}
2024-06-13 19:45:21,637 WARNING MsgRouterThr:2419930 [router.py:message_loop():77] message_loop has been closed
